app/src/androidTest/java/ch/sdp/vibester/UtilTest.kt

```diff
@@ -1,10 +1,9 @@
 package ch.sdp.vibester
 
 import ch.sdp.vibester.util.Util
+import org.junit.Assert.assertEquals
```

Comment:
Can you remove Util.kt and UtilTest since we dont generate a new id for users anymore? thanks!

Reply:
+1